src/MSBuild/Resources/Strings.resx

```diff
@@ -1154,6 +1170,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <value>MSBUILD : error MSB1047: File to preprocess to is not valid. {0}</value>
     <comment>{StrBegin="MSBUILD : error MSB1047: "}</comment>
   </data>
+  <data name="InvalidTargetsPath">
+    <value>MSBUILD : error MSB1047: File to print targets to is not valid. {0}</value>
```

Comment:
I know you copied the style of the error for preprocessing, but I don't think that message is perfect. Would you mind changing to something simple like:
`Targets could not be printed.`
Also, you reused MSB1047. Can you change the number?

Reply:
committed